redhat/release/update-to-head.sh

```diff
@@ -67,8 +67,17 @@ if [[ -d redhat/patches ]] && [ "$(ls -A redhat/patches)" ]; then
   git apply redhat/patches/*
 fi
 
-git mv redhat/overlays/log_server/Dockerfile.logserver .
-git mv redhat/overlays/log_signer/Dockerfile.logsigner .
+
+# RHTAP writes its pipeline files to the root of ${redhat_ref}
+# Fetch those from origin and apply them to the the release branch
+# since we just wiped out our local copy with the upstream ref.
+git fetch origin $redhat_ref
+git checkout origin/$redhat_ref .tekton
```

Comment:
A minor concern, but this section should probably be above line 67 so that in case we are patching anything in the .tekton directory, it gets applied.